state-chain/runtime/src/runtime_apis.rs

```diff
@@ -324,15 +324,6 @@ decl_runtime_apis!(
 			amount: AssetAmount,
 			additional_limit_orders: Option<Vec<SimulateSwapAdditionalOrder>>,
 		) -> Result<SimulatedSwapInformation!["1.0.0"], DispatchErrorWithMessage>;
-		#[changed_in(3)]
-		fn cf_pool_simulate_swap(
```

Comment:
FYI I think you need to keep this - it's not a breaking change in json, but it *is* breaking in SCALE :/